### DISCUSSION TITLE
[BUG] [SP-API] Send to Amazon will replace the send/replenish workflow for API-created shipments, but there are no SP APIs that match the Send to Amazon workflow

### DISCUSSION THREAD

**Discussion by NancyLin** [amzn/selling-partner-api-models#2828] [General] [answered]
Hello,

We received the notice that send to Amazon will replace the send/replenish workflow for API-created shipments after March 1, 2023. But we do not find the SP APIs that match the "Send to Amazon" workflow.

Now we use the fulfillment inbound API to create or update shipments. If we want to change to  "send to Amazon" workflow, which API can we use?

**Comment by GJ-JG** [comment, marked as answer]
Hi @NancyLin , you can change API-created shipment to "Send To Amazon" workflow by using Send To Amazon in Seller Central. Because once an API-created shipment is viewed/edited via Send To Amazon, it automatically became Send To Amazon workflow. 
But at the same time, please keep in mind that once a shipment that have been created, viewed, or edited using Send to Amazon, it cannot be viewed or edited via API. This includes shipments that have been created or edited via API in the past.

We will support this feature in the future and send the announcement once the launch date is decided.

**Reply by keithbrink** [reply]
@GJ-JG Is Amazon planning future support for viewing/editing Send To Amazon shipments via the API?

**Reply by GJ-JG** [reply]
> @GJ-JG Is Amazon planning future support for viewing/editing Send To Amazon shipments via the API?

Yes we do, but the ETA is not fixed yet and we will send the announcement once the launch date is decided.

**Reply by kirahome-eng1** [reply]
Hi @GJ-JG ~ Wondering if there's any updates on the ETA? We write our own in-house tool to centrally manage inbound shipments. March 1st is in two weeks, by then, as soon as user opens the shipments in seller central / STA, our tool will lose editing capability to those shipments...

**Reply by wangzhen1990** [reply]
@GJ-JG Is there any further time

**Reply by Frankfrankenberg** [reply]
Has anyone had any luck solving this issue. The old workflow saved us hours packing each week, open to any solution you may have.

**Comment by wangzhen1990** [comment]
@GJ-JG Is there any further time

**Comment by freedom64life** [comment]
@GJ-JG  is there any ETA for this function?

**Comment by teknetik** [comment]
+1 for this feature. Feels pretty bad Amazon rip out functionality without anytime of replacement. The famous API company with little to no shipping API?
